smoke/LeafDevice/details/Details.cs

```diff
@@ -156,6 +159,34 @@ protected async Task VerifyDataOnIoTHub()
             await eventHubClient.CloseAsync();
         }
 
+        static Task<MethodResponse> DirectMethod(MethodRequest methodRequest, object userContext)
+        {
+            Console.WriteLine("Leaf device received direct method call...");
```

Comment:
While we are testing it, maybe we should pass a payload and return a payload here. That will be a better check imo..

Reply:
Good idea. Done. Testing. 

---
In reply to: [219918895](https://github.com/Azure/iotedge/pull/344#discussion_r219918895) [](ancestors = 219918895)